opencog/persist/file/fast_load.h

```diff
@@ -25,11 +25,26 @@
 
 #ifndef FAST_LOAD_H
 #define FAST_LOAD_H
+
+#include <fstream>
+#include <iostream>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+#include <opencog/atoms/atom_types/NameServer.h>
+#include <opencog/atoms/base/Link.h>
+#include <opencog/atoms/base/Node.h>
+#include <opencog/atoms/core/NumberNode.h>
+#include <opencog/atoms/truthvalue/SimpleTruthValue.h>
+#include <opencog/atomspace/AtomSpace.h>
```

Comment:
Can you move these header files back to the .cc file? There is nothing in this header file that needs any of these; they don't belong here ... they belong only in the c code that actually needs them.